resources/install.rb

```diff
@@ -24,6 +24,7 @@
   remote_file 'install nssm' do
     path ::NSSM.binary_path node
     source "file:///#{system_file}"
+    only_if {::File.exist?("#{Chef::Config[:file_cache_path]}/#{basename}/#{system}/nssm.exe")}
```

Comment:
Why is it notification friendly?
Notification are bypassing only_if/not_if guards.

In a separate PR, could you explain clearly the initial bug (I didn't encounter it) and the resolution?

Reply:
It fixes #33 let me know if enough details are in there. It only happens when you use a before notification, the other types work.

I am trying to see how to split the commits into separate PR's and submit them separately but my GIT isn't very strong.